cmd/lakefs/cmd/migrate.go

```diff
@@ -115,10 +113,8 @@ func DoMigration(ctx context.Context, kvStore kv.Store, cfg *config.Config, forc
 		switch {
 		case version >= kv.NextSchemaVersion || version < kv.InitialMigrateVersion:
 			return fmt.Errorf("wrong starting version %d: %w", version, kv.ErrMigrationVersion)
-		case version < kv.ACLNoReposMigrateVersion:
-			err = migrations.MigrateToACL(ctx, kvStore, cfg, logging.ContextUnavailable(), version, force)
-		case version < kv.ACLImportMigrateVersion:
-			err = migrations.MigrateImportPermissions(ctx, kvStore, cfg)
+		default: // ACL migration code is no longer needed since we removed ACLs
+			err = kv.SetDBSchemaVersion(ctx, kvStore, kv.ACLImportMigrateVersion)
```

Comment:
It feels a bit wrong making `kv.ACLImportMigrateVersion` a version that could be anything. 
I do understand that adding a new version will require a useless migration.
Not requesting anything, just thinking out loud

Reply:
We could create a new version and require upgrade - but that would make the nocal version officially a migration version and require users to actively run migrate (which we wanted to avoid)